### ISSUE TITLE
Create AI interaction cards

### ISSUE THREAD

**Issue by briangormanly** [agorafoundation/agora#546]

**Comment by jaymehta002** [comment]
can you share more details?

**Comment by briangormanly** [comment]
@jaymehta002 Yes, apologies just building out the tickets for the project, I currently thinking through the Design, but essentially the panel that I just added in the New-AI-Main that slides up from the bottom of the screen 
<img width="1281" alt="Screenshot 2024-01-10 at 11 40 12 AM" src="https://github.com/agorafoundation/agora/assets/2549569/cd24b14b-50af-41ad-8720-6f990b884ae4">
Will have cards that will show up in the panel that will contain different AI suggestions these might be for content, additional sections, references, grammar, etc) The content for the cards and when they will be created will be managed by the AI interaction engine #544 . There are many current cards used throughout Agora, the dashboard has cards for workspaces for example:
<img width="1407" alt="Screenshot 2024-01-10 at 11 55 56 AM" src="https://github.com/agorafoundation/agora/assets/2549569/ea705f23-7660-41c9-8afc-a205466d5677">
And the friends panel as well:
<img width="1411" alt="Screenshot 2024-01-10 at 11 56 39 AM" src="https://github.com/agorafoundation/agora/assets/2549569/2cce07f0-2b0c-4052-b3b6-ea03cd84f75e">
I don't know if either of those are necessarily a perfect fit, but would be a start.

As I am writing this it is helping me think through some of the requirements. I am going to create a wiki article with more detail for this project and link it back when done. Thank you for the interest!

**Comment by jaymehta002** [comment]
i can re-design the ui if you want.

**Comment by jaymehta002** [comment]
i cannot understand the problem.
you want me to redesign the frontend or add some AI features

**Comment by briangormanly** [comment]
Sure, I am still building out the design, as soon as I have it done I will let you know. Then we can see if you still have questions.

**Comment by jaymehta002** [comment]
okay
